internal/postgres/config.go

```diff
@@ -99,6 +100,54 @@ func Environment(cluster *v1beta1.PostgresCluster) []corev1.EnvVar {
 	}
 }
 
+// reloadCommand returns an entrypoint that convinces PostgreSQL to reload
+// certificate files when they change. The process will appear as name in `ps`
```

Comment:
>The process will appear as name in `ps`

Should this be something other than name :question:

Reply:
This is what it looks like with the wrapper. Without it, the entire multi-line script appears: `bash -c declare …`

```console
$ kubectl exec pod/mycluster-00-bvw9-0 -- ps ux
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
postgres      39  0.2  0.8 606752 33128 ?        Ssl  21:11   0:01 /usr/bin/python3.6 /usr/local/bin/patroni /etc/patroni
postgres      46  0.0  0.0  14844  3196 ?        Ss   21:11   0:00 replication-cert-copy -ceu monitor
…
```